zypp/base/CleanerThread.cc

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------\
+|                          ____ _   __ __ ___                          |
+|                         |__  / \ / / . \ . \                         |
+|                           / / \ V /|  _/  _/                         |
+|                          / /__ | | | | | |                           |
+|                         /_____||_| |_| |_|                           |
+|                                                                      |
+\---------------------------------------------------------------------*/
+/** \file	zypp/base/CleanerThread.cc
+ */
+
+#include "zypp/base/CleanerThread_p.h"
+#include <algorithm>
+#include <thread>
+#include <mutex>
+#include <condition_variable>
+#include <chrono>
+#include <vector>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+struct CleanerData
+{
+  static CleanerData &get ()
```

Comment:
ZYpp singletons foll the convention to name the access method `instance`.

Reply:
done